### PR TITLE
Reformat and update error strings.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by jsafrane** [comment]
The right error format is `fmt.Errorf("error doing something with %s: %s", something.Name, err)`. Like fmt.Errorf(`failed to get endpoint %s: %s", epName, err)`

+please add namespace to endpoints and services

**Comment by humblec** [comment]
@jsafrane Changed the log format to above suggested model. Thanks !!

**Comment by humblec** [comment]
@jsafrane PTAL :)

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -984,7 +987,7 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 				parseVolumeNamePrefix = v
 			}
 		default:
-			return nil, fmt.Errorf("invalid option %q for volume plugin %s", k, glusterfsPluginName)
+			return nil, fmt.Errorf("invalid option [%q] for volume plugin %s", k, glusterfsPluginName)
```
Comment (jsafrane) [inline pkg/volume/glusterfs/glusterfs.go]: No `[]` around please, there already are double quotes.

File: pkg/volume/glusterfs/glusterfs.go [hunk 2]
```diff
@@ -1002,7 +1005,7 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 			if len(parseVolumeTypeInfo) >= 2 {
 				newReplicaCount, err := convertVolumeParam(parseVolumeTypeInfo[1])
 				if err != nil {
-					return nil, fmt.Errorf("error [%v] when parsing value %q of option '%s' for volume plugin %s", err, parseVolumeTypeInfo[1], "volumetype", glusterfsPluginName)
+					return nil, fmt.Errorf("error %v when parsing value %q of option '%s' for volume plugin %s", err, parseVolumeTypeInfo[1], "volumetype", glusterfsPluginName)
```
Comment (jsafrane) [inline pkg/volume/glusterfs/glusterfs.go]: `"error parsing volumeType %q: %s", parseVolumeTypeInfo[1], err"

IMO, the plugin name is not relevant.

(applies to all volumeType errors below)

File: pkg/volume/glusterfs/glusterfs.go [hunk 3]
```diff
@@ -441,7 +443,7 @@ func convertVolumeParam(volumeString string) (int, error) {
 
 	count, err := strconv.Atoi(volumeString)
 	if err != nil {
-		return 0, fmt.Errorf("failed to parse %q", volumeString)
+		return 0, fmt.Errorf("failed to parse volumestring [%q]: %v", volumeString, err)
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: Done.

File: pkg/volume/glusterfs/glusterfs.go [hunk 4]
```diff
@@ -149,18 +150,18 @@ func (plugin *glusterfsPlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, _ volu
...
 		return nil, fmt.Errorf("failed to get kube client to initialize mounter")
 	}
 	ep, err := kubeClient.CoreV1().Endpoints(podNs).Get(epName, metav1.GetOptions{})
 	if err != nil {
-		glog.Errorf("failed to get endpoints %s[%v]", epName, err)
+		glog.Errorf("failed to get endpoint %s: %v", epName, err)
 		return nil, err
 	}
-	glog.V(1).Infof("glusterfs pv endpoint: [%v]", ep)
+	glog.V(1).Infof("glusterfs pv endpoint %v", ep)
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: Done

File: pkg/volume/glusterfs/glusterfs.go [hunk 5]
```diff
@@ -658,26 +659,26 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 	if pvSpec.Glusterfs.EndpointsName != "" {
 		dynamicEndpoint = pvSpec.Glusterfs.EndpointsName
 	}
-	glog.V(3).Infof("dynamic namespace and endpoint : [%v/%v]", dynamicNamespace, dynamicEndpoint)
+	glog.V(3).Infof("dynamic namespace and endpoint %v%v", dynamicNamespace, dynamicEndpoint)
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: Done.

File: pkg/volume/glusterfs/glusterfs.go [hunk 6]
```diff
@@ -658,26 +659,26 @@ func (d *glusterfsVolumeDeleter) Delete() error {
...
 	err = d.deleteEndpointService(dynamicNamespace, dynamicEndpoint)
 	if err != nil {
-		glog.Errorf("error when deleting endpoint/service :%v", err)
+		glog.Errorf("failed to delete endpoint/service %v/%v: %v", dynamicNamespace, dynamicEndpoint, err)
 	} else {
-		glog.V(1).Infof("endpoint: [%v/%v] is deleted successfully ", dynamicNamespace, dynamicEndpoint)
+		glog.V(1).Infof("endpoint %v%v is deleted successfully ", dynamicNamespace, dynamicEndpoint)
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: Done.

File: pkg/volume/glusterfs/glusterfs.go (hunk 1 above)
Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: Done.

File: pkg/volume/glusterfs/glusterfs.go (hunk 2 above)
Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: Done.

**Comment by humblec** [comment]
@jsafrane addressed the comments, Thanks !

**Pull request merged**